src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java

```diff
@@ -220,6 +228,31 @@ public void testVerifyValue() throws Exception {
                 }).get();
     }
 
+    @Test
+    public void testEnumValue() throws Exception {
+        TestHelper.dropAllSchemas();
+        TestHelper.executeDDL("yugabyte_create_tables.ddl");
+        Thread.sleep(1000);
+
+        String dbStreamId = TestHelper.getNewDbStreamId("yugabyte", "test_enum");
+        Configuration.Builder configBuilder = TestHelper.getConfigBuilder("public.test_enum", dbStreamId);
+        start(YugabyteDBConnector.class, configBuilder.build());
+        assertConnectorIsRunning();
+
+        // 3 because there are 3 enum values in the enum type
+        final long recordsCount = 3;
+
+        awaitUntilConnectorIsReady();
+
+        // 3 records will be inserted in the table test_enum
+        insertEnumRecords();
+
+        CompletableFuture.runAsync(() -> verifyPrimaryKeyOnly(recordsCount))
```

Comment:
Is there an assert to ensure enum value is correct in the source record?

Reply:
We can add one anytime.